### DISCUSSION TITLE
How to exclude empty label value from search result

### DISCUSSION THREAD

**Discussion by WPum** [TriliumNext/Trilium#1825] [Q&A] [answered]
I want to exclude #myLabel (empty value) from search result.
Search for `#myLabel` simply gives me all notes with #myLabel and #myLabel=myValue
`#myLabel != ''` and `note.labels.myLabel != ''` don't work.

**Comment by zadam** [comment, marked as answer]
Hi, `#myLabel != ''` is correct, but you stumbled upon a bug in parsing empty strings. This was fixed in stable branch and will be released in the next patch release.

Thanks for reporting!
